src/style.css

```diff
@@ -1,3 +1,81 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+ul,
+ol {
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+button {
+  cursor: pointer;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.moyo_header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.moyo_menu {
+  display: flex;
+  list-style: none;
+  padding: 0;
```

Comment:
The `.moyo_menu` class has `list-style: none;` which is good, but the same rule is redundantly applied to the `ul` tag on line 12. Consider removing the tag selector if all lists in the project are meant to have no list-style.